modules/sso/config/launch/setup/73/added/standalone-openshift.xml

```diff
@@ -130,8 +130,8 @@
                     <driver name="h2" module="com.h2database.h2">
                         <xa-datasource-class>org.h2.jdbcx.JdbcDataSource</xa-datasource-class>
                     </driver>
-                    <driver name="mysql" module="com.mysql">
-                        <xa-datasource-class>com.mysql.jdbc.jdbc2.optional.MysqlXADataSource</xa-datasource-class>
+                    <driver name="mariadb" module="org.mariadb">
+                        <xa-datasource-class>org.mariadb.jdbc.MySQLDataSource</xa-datasource-class>
```

Comment:
Needs to be `org.mariadb.jdbc.MariaDbDataSource` per https://mariadb.com/kb/en/library/about-mariadb-connector-j/#optional-jdbc-classes

Reply:
@hmlnarik Great catch, thanks! Updated / fixed & squashed into previous change.